docs/getting-started/gs-faqs.md

```diff
@@ -172,8 +183,8 @@ The system should follow the [email normalization rules](gs-normalization-encodi
 
 Yes. Not storing mappings may increase processing time drastically when you have to map millions of email addresses or phone numbers. Recalculating only those mappings that actually need to be updated, however, reduces the total processing time because only about 1/365th of UID2s need to be updated daily.
 
-:::info
-Unless you are using a <Link href="../ref-info/glossary-uid#gl-private-operator">Private Operator</Link>, you must map email addresses, phone numbers, or hashes consecutively, using a single HTTP connection, in batches of 5,000 emails at a time. In other words, do your mapping without creating multiple parallel connections. 
+:::important
+Unless you are using a <Link href="../ref-info/glossary-uid#gl-private-operator">Private Operator</Link>, you must map email addresses, phone numbers, or hashes consecutively, using a single HTTP connection, in batches of 5,000 emails at a time. In other words, do your mapping without creating multiple parallel connections.
```

Comment:
I know this was there before, but curious if the should be changed to: "in batches of at most 5,000 emails or phone numbers at a time":
- add mention of other input types, not just emails
- indicate that 5,000 is a cap, not a required number

Reply:
Yes, thx! Used wording we had elsewhere (in POST /identity/map): "with a maximum batch size of 5,000 items per batch"
BTW happy for any comments from you on any of the docs at any point.